### ISSUE TITLE
Does blaze support set difference operation?

### ISSUE THREAD

**Issue by HereticSK** [blaze/blaze#1604]
Some times I need to fetch data as a result set of set difference.
In SQL I write:
```SQL
SELECT date, symbol FROM table1
EXCEPT
SELECT date, symbol FROM table2
```
Is there any way I can do this with blaze? I find the syntax of blaze extremely useful, that I want to build a data fetcher with it to avoid writing trivial SQL. The expression of blaze should satisfy most of my need. Set difference operation is the only missing piece.

**Comment by llllllllll** [comment]
I don't believe this is currently supported; however, we could definitly implement it.
